test/sub_server/test_simulator_api.py

```diff
@@ -0,0 +1,467 @@
+"""Test simulator API."""
```

Comment:
This is not right !! it is as if you have rewritten the whole file.

Reply:
What do you mean? It's a new file. Did you

1. Confuse it with test_simulator.py?
2. Want me to combine it with it?

Reply:
Seems I got confused.